src/components/UseAuth.js

```diff
@@ -0,0 +1,18 @@
+import { useState } from "react";
+
+export default function useAuth(initialValue) {
+  const [isAuth, setIsAuth] = useState(initialValue);
```

Comment:
initialValue ?

Should that be `false` every time when initialized ?

Reply:
Corrected